comp/radio-plan.yaml

```diff
@@ -0,0 +1,33 @@
+summary: Plan who will be using the radios at the competition
+
+priority: should
```

Comment:
If radios are required for the incident response plan, perhaps this is a "must"? Or is the idea that we change the plan instead?

Reply:
This depends on what the incident response plan decides.